packages/storefront/src/components/Markdown.vue

```diff
@@ -259,6 +259,9 @@
           border-radius: $pds-border-radius-medium;
           overflow-x: auto;
           -webkit-overflow-scrolling: touch;
+          &:focus {
```

Comment:
The `pds-focus()` mixin comes already with `:focus` and `:focus-visible` pseudo classes. So no need to nest additional `:focus` which would render to `pre:focus:focus` in generated code.